docs/pages/jupyter-notebooks.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Jupyter Notebooks
+layout: default
+---
+
+# Jupyter notebooks
+
+We generally recommend developing code for packaging into python modules if possible.
+However occasionally we support researchers who prefer a notebook environment or projects which want to provide examples and tutorials in notebook format.
```

Comment:
```suggestion
We generally recommend packaging resuable code components into Python modules where possible.
However occasionally we support researchers who prefer a notebook environment or projects which want to provide examples and tutorials in notebook format.
Notebooks can also be a valid alternative to Python scripts for running and recording the results of numerical experiments for example.
```

I personally think of notebooks more of as an alternative to writing a script which runs some analysis but with the benefit of allowing richer Markdown notes and outputs like plots to be integrated alongside the code. Might be on my own here though!